go/cmd/vttestserver/main.go

```diff
@@ -96,75 +95,79 @@ func init() {
 			" migrations are run every time the cluster starts, since persistence"+
 			" for the topology server has not been implemented yet")
 
-	flag.BoolVar(&doSeed, "initialize_with_random_data", false,
+	fs.BoolVar(&doSeed, "initialize_with_random_data", false,
 		"If this flag is each table-shard will be initialized"+
 			" with random data. See also the 'rng_seed' and 'min_shard_size'"+
 			" and 'max_shard_size' flags.")
 
-	flag.IntVar(&seed.RngSeed, "rng_seed", 123,
+	fs.IntVar(&seed.RngSeed, "rng_seed", 123,
 		"The random number generator seed to use when initializing"+
 			" with random data (see also --initialize_with_random_data)."+
 			" Multiple runs with the same seed will result with the same"+
 			" initial data.")
 
-	flag.IntVar(&seed.MinSize, "min_table_shard_size", 1000,
+	fs.IntVar(&seed.MinSize, "min_table_shard_size", 1000,
 		"The minimum number of initial rows in a table shard. Ignored if"+
 			"--initialize_with_random_data is false. The actual number is chosen"+
 			" randomly.")
 
-	flag.IntVar(&seed.MaxSize, "max_table_shard_size", 10000,
+	fs.IntVar(&seed.MaxSize, "max_table_shard_size", 10000,
 		"The maximum number of initial rows in a table shard. Ignored if"+
 			"--initialize_with_random_data is false. The actual number is chosen"+
 			" randomly")
 
-	flag.Float64Var(&seed.NullProbability, "null_probability", 0.1,
+	fs.Float64Var(&seed.NullProbability, "null_probability", 0.1,
 		"The probability to initialize a field with 'NULL' "+
 			" if --initialize_with_random_data is true. Only applies to fields"+
 			" that can contain NULL values.")
 
-	flag.StringVar(&config.MySQLBindHost, "mysql_bind_host", "localhost",
+	fs.StringVar(&config.MySQLBindHost, "mysql_bind_host", "localhost",
 		"which host to bind vtgate mysql listener to")
 
-	flag.StringVar(&mycnf, "extra_my_cnf", "",
+	fs.StringVar(&mycnf, "extra_my_cnf", "",
 		"extra files to add to the config, separated by ':'")
 
-	flag.StringVar(&topo.cells, "cells", "test", "Comma separated list of cells")
-	flag.StringVar(&topo.keyspaces, "keyspaces", "test_keyspace",
+	fs.StringVar(&topo.cells, "cells", "test", "Comma separated list of cells")
```

Comment:
TOILI: if you switch the type of `topo.cells` to `[]string` you can use `fs.StringSliceVar` and remove our own `strings.Split` code (and similar for all the other `StringVar` flags that are comma-separated)

Reply:
cells, keyspaces and strings. All three were accepting comma separated list of strings. I ported them all over to StrilgSliceVar.